src/coreclr/tests/src/tracing/eventpipe/stoponstart/stoponstart.cs

```diff
@@ -0,0 +1,197 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using System.Reflection;
+using Microsoft.Diagnostics.Tools.RuntimeClient;
+using Tracing.Tests.Common;
+using System.Threading;
+using System.IO;
+using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+
+namespace Tracing.Tests.StopOnStartValidation
+{
+    public class StopOnStartValidation
+    {
+        public static async Task<bool> TEST_RuntimeResumesExecutionWithCommand()
+        {
+            string serverName = ReverseServer.MakeServerAddress();
+            Logger.logger.Log($"Server name is '{serverName}'");
+            var server = new ReverseServer(serverName);
+            Task subprocessTask = RunSubprocess(
+                serverName: serverName,
+                duringExecution: async (_) =>
+                {
+                    Stream stream = await server.AcceptAsync();
+                    IpcAdvertise advertise = IpcAdvertise.Parse(stream);
+                    Logger.logger.Log(advertise.ToString());
+                    // send ResumeRuntime command (2=EventPipeCommandSet, 4=ResumeRuntime commandid)
+                    var message = new IpcMessage(2,4);
+                    IpcMessage response = IpcClient.SendMessage(stream, message);
+                    Logger.logger.Log(response.ToString());
+                }
+            );
+
+            await WaitTillTimeout(subprocessTask, TimeSpan.FromMinutes(1));
+
+            return true;
+        }
+
+        public static async Task<bool> TEST_TracesHaveRelevantEvents()
```

Comment:
You may also want to test a variation: if you don't resume the runtime then we should not have events.